romanian/sfarsit/the_end.md

```diff
@@ -0,0 +1,4 @@
+Dupa ce te trezesti, simti de parca ai pierdut o oportunitate uriasa.
```

Comment:
If you move the directory `sfarsit` and its files inside of the `somn` directory, that should fix the link, and I think it's also a better organization.